deps/src/main/java/com/microsoft/azure/sdk/iot/deps/twin/TwinMetadata.java

```diff
@@ -164,14 +187,43 @@ else if(key.equals(LAST_UPDATE_VERSION_TAG))
                 }
                 lastUpdatedVersion = ((Number)entry.getValue()).intValue();
             }
+            else if (key.equals(TwinMetadata.LAST_UPDATED_BY))
+            {
+                lastUpdatedBy = (String)entry.getValue();
+            }
+            else if (key.equals(TwinMetadata.LAST_UPDATED_BY_DIGEST))
+            {
+                lastUpdatedByDigest = (String)entry.getValue();
+            }
         }
+        // TODO: additional checks
```

Comment:
awaiting response from service team to understand if we need to check for null values in newly added properties.